Makefile

```diff
@@ -12,9 +12,18 @@ REPOS_PATH=repos
 _SYSUPG_REPOS="$(REPOS_PATH)/system_upgrade"
 LIBRARY_PATH=
 REPORT_ARG=
-REPOSITORIES ?= $(shell ls $(_SYSUPG_REPOS) | xargs echo | tr " " ",")
-SYSUPG_TEST_PATHS=$(shell echo $(REPOSITORIES) | sed -r "s|(,\\|^)| $(_SYSUPG_REPOS)/|g")
-TEST_PATHS:=commands repos/common $(SYSUPG_TEST_PATHS)
+
+ifdef ACTOR
+	# If REPOSITORIES is set, search for the actor only in the specified repos,
+	# if REPOSITORIES is not set i.e. it's empty, search in all repos, this is
+	# broken due to name collisions in repositories (FIXME)
```

Comment:
```suggestion
	# If REPOSITORIES is set, the `utils/actor_path.py` script searches for the actor only in the specified repos.
	# if REPOSITORIES is not set i.e. it's empty, all repositories are searched -- this is
	# broken due to name collisions in repositories (FIXME)
```

Reply:
This will make it a bit clearer that the script `utils/actor_path.py` has the last argument optional (if it is missing, it will search everything)